models/staging/tpdm/base/base_tpdm__post_secondary_institutions.sql

```diff
@@ -0,0 +1,25 @@
+with post_secondary_institutions as (
+    {{ source_edfi3('post_secondary_institutions') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```